mantidproject/autoreduce
# Moved logging and other settings out of critical scripts

Fixes #209 

To Test:

* On the autoreducedev server confirm that messages are working as expected (see https://github.com/mantidproject/autoreduce/pull/180)
* Modify /bin/logging_setup.py to have a different logging level
* Reset the queueProcessor_daemon
* Confirm the logging level has changed for both PostProcessAdmin and queueProcessor
